plugins/completion/cmp/default.nix

```diff
@@ -16,6 +16,43 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   maintainers = [ maintainers.GaetanLepage ];
 
+  description = ''
+    ### Completion Source Installation
+
+    If `plugins.cmp.autoEnableSources` Nixivm will automatically enable the corresponding source
+    plugins. This will work only when this option is set to a list.
+    If you use a raw lua string, you will need to explicitly enable the relevant source plugins in
+    your nixvim configuration.
+
+    ```nix
```

Comment:
Maybe add a small caption to tell the difference in the two blocks, something like "With auto-enabled sources", "Without auto-enabled sources" ?